test/fixtures/dummy_feats/feature_manifest.yml

```diff
@@ -15,46 +15,49 @@ feature_extractor:
   spectrogram_config:
     dither: 0.0
     energy_floor: 0.0
-    frame_length: 25.0
-    frame_shift: 10.0
+    frame_length: 0.025
```

Comment:
can we comment that these will be converted to milliseonds before Kaldi sees them?  not sure if yaml has comments?